.github/workflows/build.yml

```diff
@@ -24,24 +24,30 @@ jobs:
           - "6.0"
           - "6.1"
           - main
+        include:
```

Comment:
I'd prefer to keep the existing pattern of adding the new Rails and Ruby versions to the matrix section above, then excluding the combinations we don't want. That has generally made it slightly easier to add new versions of things.

Reply:
After running the workflow I see we're missing a bunch of combinations, so all the more reason to stick to the existing matrix approach.